src/lib/store-cash/store-cash-fpti.js

```diff
@@ -0,0 +1,149 @@
+/* @flow */
+import { getMerchantID } from '@paypal/sdk-client/src';
+
+import type { Config, StoreCashVariables } from '../../types';
+import { sendBeacon, filterFalsyValues } from '../fpti';
+import {
+  getPageTitle,
+  getBrowserPlugins,
+  getWindowLocation,
+  getDeviceInfo
+} from '../get-device-info';
+
+const resolveTrackingData = (config, data) => {
+  const storeCashProgramId = config.containerSummary && config.containerSummary.storeCashProgramId;
+
+  const deviceInfo = getDeviceInfo();
+  const completeUrl = getWindowLocation();
+  const browserPlugins = getBrowserPlugins();
+  const pageTitle = getPageTitle();
+  const mrid = getMerchantID()[0];
+
+  return {
+    ...deviceInfo,
+    ...config,
+    ...data,
+    comp: 'tagmanagernodeweb',
+    tsrce: 'tagmanagernodeweb',
+    subcomponent: 'sdk',
```

Comment:
thus could present a problem if the data pipeline is filtering for a specific subcomponent